Packs/Snowflake/Integrations/Snowflake/Snowflake_test.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+from CommonServerPython import *
+
+def test_convert_datetime_to_string(mocker):
```

Comment:
add tests for a couple more formats? especially the 1s that we saw in this ticket